src/settings-ui/Settings.UI/Helpers/NativeKeyboardHelper.cs

```diff
@@ -18,7 +18,7 @@ internal struct INPUT
 
             internal static int Size
             {
-                get { return Marshal.SizeOf(typeof(INPUT)); }
+                get { return Marshal.SizeOf<INPUT>(); }
```

Comment:
Is `INPUT` blittable? If so you should just do `sizeof(INPUT)`. Same for all other `Marshal.SizeOf` uses in this PR.